README.md

```diff
@@ -1,3 +1,29 @@
 # bids-methods
 
 Repository for templates to generate methods section descriptions of BIDS datasets
+
+These templates use the Mustache templating language: https://mustache.github.io/
+
+## Code style
+
+When some BIDS metadata is used "as is" as a mustache variable,
+we kept its original name and use CamelCase.
+
+```mustache
+The data acquisition was performed in the {{ InstitutionName }}, {{ InstitutionalDepartmentName }}, {{ InstitutionAddress }}.
+```
+
+For some other metadata eventhough we changed their name for the mustache variable.
+For some this was to enforce using a more appropriate unit for methods reporting.
+
+For example we do:
+
+```mustache
+repetition time, TR= {{ tr }} ms;
+```
+
+instead of
+
+```mustache
+repetition time, TR= {{ RepetitionTime }} seconds;
+```
```

Comment:
probably worth checking all templates to implement this more systematically

also would need to change back some variable names: 
for example `flip_angle` should be `FlipAngle`